packages/zoe/src/contracts/loan/updateDebt.js

```diff
@@ -38,6 +35,11 @@ export const makeDebtCalculator = debtCalculatorConfig => {
     configMinusGetDebt,
   } = debtCalculatorConfig;
   let debt = originalDebt;
+  const interestRatio = makeRatio(
```

Comment:
Huh, I'm thinking that `interestRate` in the terms should be `interestRatio` instead and this should be moved outside the contract. What do you think?

Reply:
Ok that sounds fine to me

Reply:
@Chris-Hibbert were you planning on still moving `interestRatio = makeRatio(` out of the contract? If so, should I come back and continue my review then? Everything else looks good to me.

Reply:
This is now fixed.